app/controllers/return-requirements.controller.js

```diff
@@ -10,6 +10,7 @@ const AddService = require('../services/return-requirements/add.service.js')
 const AgreementsExceptionsService = require('../services/return-requirements/agreements-exceptions.service.js')
 const CancelService = require('../services/return-requirements/cancel.service.js')
 const CheckService = require('../services/return-requirements/check.service.js')
+const AdditionalSubmissionOptionsService = require('../services/return-requirements/additional-submission-options.service.js')
```

Comment:
NIT - think this just needs to go up a few lines to be under `AbstractionPeriodService`.